### PR TITLE
Feature/checkin admin ability

### PR DESCRIPTION
**Pull request by jimmylevell**
Implemented 
- new admin ability for checkin checkout process to allow export
- new tn admin reader role
- display assigned units on activity view

### CODE REVIEW

**Comment by jimmylevell** [comment]
todo;

- [ ] inform brain regarding new role

File: app/models/ability.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def initialize(user)
 
     admin_user_permissions(user) if user.role_admin?
     checkin_checkout_user_permissions(user) if user.role_checkin_checkout?
+    checkin_checkout_user_admin_permissions(user) if user.role_checkin_checkout_admin?
```
Comment (sihu) [inline app/models/ability.rb]: maybe admin is a bit misleading? can we cal it `checkin_checkout_manager?`?

File: app/models/user.rb [hunk 2]
```diff
@@ -20,7 +20,7 @@
 #
 class User < ApplicationRecord
   ROLES = %i[user admin programm tn_administration editor allocation read_unit unit_communication
-             checkin_checkout ].freeze
+             checkin_checkout checkin_checkout_admin ].freeze
```
Comment (sihu) [inline app/models/user.rb]:
```suggestion
             checkin_checkout checkin_checkout_admin].freeze
```

**Review by sihu — APPROVED** [review]

File: app/models/ability.rb (hunk 1 above)
Comment (jimmylevell) [reply app/models/ability.rb]: totally agree! changed

**Comment by jimmylevell** [comment]
- [x] code review
- [x] publish prod
- [x] inform Brain regarding new roles

File: app/models/ability.rb [hunk 3]
```diff
@@ -10,7 +10,9 @@ def initialize(user)
 
     admin_user_permissions(user) if user.role_admin?
     checkin_checkout_user_permissions(user) if user.role_checkin_checkout?
+    checkin_checkout_user_manager_permissions(user) if user.role_checkin_checkout_manager?
```
Comment (sihu) [inline app/models/ability.rb]:
```suggestion
    checkin_checkout_manager_user_permissions(user) if user.role_checkin_checkout_manager?
```

File: app/models/ability.rb [hunk 4]
```diff
@@ -34,6 +36,11 @@ def checkin_checkout_user_permissions(_user)
     can %i[create update read], CheckpointUnit
   end
 
+  def checkin_checkout_user_manager_permissions(_user)
```
Comment (sihu) [inline app/models/ability.rb]:
```suggestion
  def checkin_checkout_manager_user_permissions(_user)
```

Comment (sihu) [inline app/models/ability.rb]:
```suggestion
  def checkin_checkout_manager_user_permissions(_user)
```

File: app/models/ability.rb [hunk 5]
```diff
@@ -34,6 +36,11 @@ def checkin_checkout_user_permissions(_user)
...
+    can %i[manage], Checkpoint
```
Comment (sihu) [inline app/models/ability.rb:40]: Do they need to be able to change all checkpoints? There is also a export ability
```suggestion
    can %i[export], CheckpointUnit
```

File: app/models/ability.rb [hunk 6]
```diff
@@ -10,7 +10,9 @@ def initialize(user)
 
     admin_user_permissions(user) if user.role_admin?
     checkin_checkout_user_permissions(user) if user.role_checkin_checkout?
+    checkin_checkout_user_manager_permissions(user) if user.role_checkin_checkout_manager?
     tn_administration_user_permissions(user) if user.role_tn_administration?
+    tn_reader_permissions(user) if user.role_tn_reader?
```
Comment (sihu) [inline app/models/ability.rb]:
```suggestion
    tn_reader_user_permissions(user) if user.role_tn_reader?
```

File: app/views/activities/_filter.html.slim [hunk 7]
```diff
@@ -29,6 +29,11 @@
             = ActivityFilter.human_attribute_name(:text)
             - if @filter.text&.size&.>(2)
               .badge.rounded-pill.bg-info.ms-2= 1
+        li.nav-item.me-2.mb-2
+          a.btn.btn-outline-secondary[data-bs-toggle="pill" href="#filter-number_of_units" role="pill"]
+            = t('activity.index.numberOfAssignedUnits')
+            - if @filter.number_of_units.presence
```
Comment (sihu) [inline app/views/activities/_filter.html.slim]:
```suggestion
            - if @filter.number_of_units.present?
```

File: docker-compose.yml [hunk 8]
```diff
@@ -76,6 +77,14 @@ services:
       POSTGRES_DB: orca_e2e
       POSTGRES_PASSWORD: postgres
 
+  dbadmin:
```
Comment (sihu) [inline docker-compose.yml:80]: Hmmm...

**Review by sihu — COMMENTED** [review]
Very nice job clean up the db changes ...the rest is clarification or naming. Feel free to change if you have time. Not important

File: app/models/ability.rb (hunk 3 above)
Comment (jimmylevell) [reply app/models/ability.rb]: resolved

**Pull request merged**